submissions/yanyshpolska/Hooli-style Popup/index.html

```diff
@@ -0,0 +1,220 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="icon" type="image/x-icon" href="/img/favicon.png" />
+    <link rel="stylesheet" href="css/style.css" />
+    <title>Hooli-style Popup</title>
+  </head>
+  <body>
+    <header>
+      <nav class="menu__items">
+        <a class="menu__item_text menu__item" href="#">Mail</a>
+        <a class="menu__item_text menu__item" href="#">Images</a>
+
+        <div class="menu__item menu__item_checkbox">
+          <input
+            id="menu_checkbox"
+            class="menu__item_checkbox_input visually-hidden"
+            type="checkbox"
+          />
+          <label for="menu_checkbox" class="menu__item_checkbox_label"> </label>
+
+          <div class="popup__wrapper">
+            <div class="popup__items">
+              <a href="#" class="popup__item">
+                <div class="popup__item_image_container">
```

Comment:
you can try to remove this div or text span and reduce the amount of tags, or wrap text and img in one div and use `flex`.

Reply:
Here I use div-container-for-picture in purpose because all pictures have different sizes and this approach allows me to deal with it: regardless of the size of the picture, the menu item cells will be the same.

Reply:
You can specify image width explicitly in corresponding class for images. You can leave it, but this is far from optimal